astropy_helpers/tests/test_setup_helpers.py

```diff
@@ -1,7 +1,9 @@
 import os
 import sys
+import six
```

Comment:
Minor point, but as I recall we didn't required six before, so maybe a hack for the version number below would be slightly better.